app/src/main/res/layout/color_picker_bottom_sheet.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
+
+    <com.example.drawingApp.GradientSquare
+        android:id="@+id/gradientSquare"
+        android:layout_width="200dp"
+        android:layout_height="200dp"
+        android:layout_margin="@dimen/pad_1" />
+
+    <com.example.drawingApp.ColorSlider
+        android:id="@+id/slider"
+        android:layout_width="20dp"
+        android:layout_height="200dp"
+        android:layout_margin="@dimen/pad_1" />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="center"
+        android:orientation="vertical">
+
+        <TextView
+            android:id="@+id/colorOne"
+            android:layout_width="match_parent"
+            android:layout_height="35dp"
+            android:background="@color/black" />
+
+        <TextView
+            android:id="@+id/colorTwo"
+            android:layout_width="match_parent"
+            android:layout_height="35dp"
```

Comment:
Don't give height to the textviews in here, you may cut off text

Reply:
swapped to wrap